### DISCUSSION TITLE
Welcome to is-amaz.ing Discussions!

### DISCUSSION THREAD

**Discussion by Joabutt** [is-amazing/register#12] [Announcements]
Welcome! This is the place for you to give suggestions, and ask questions, if needed.

**Comment by QwertyIsCoding** [comment]
So... I have a doubt... How are you guys making these domains? And can you make it say anything you want? I understand a little about how you are doing this. All I know is that there is an existing .cool domain and you are adding a subdomain for that domain and when we add our site, you make a subdomain for the existing subdomain. Does that make any sense?

**Reply by Joabutt** [reply]
Hello, we are running this on a self made api which calls the cloudflare api to create the subdomain, and the self made api is called via github actions when a pull request is merged. I hope that answers your question ;)
